java-service.gitlab-ci.yml

```diff
@@ -36,6 +34,8 @@ maven_build:
     - 'mvn $MAVEN_CLI_OPTS package'
   except:
     - schedules
+  artifacts:
+    paths: [target]
```

Comment:
I don't recommend making all of target an artifact.  I would limit it to the jar or other distributable.  A lot of junk is put into target/.

then again if you use artifactory, you shouldn't need an artifact at all.

Reply:
We don't use artifactory. Perhaps we should.

Reply:
And thanks.